test/testAppinfoJsonFile.js

```diff
@@ -0,0 +1,492 @@
+            "localized_json/nl/BE",
+            "localized_json/nl",
+            "localized_json/or",
+            "localized_json/pa",
+            "localized_json/pa/PK",
+            "localized_json/pl",
+            "localized_json/pt/AO",
+            "localized_json/pt",
+            "localized_json/pt/GQ",
+            "localized_json/pt/CV",
+            "localized_json/pt/PT",
+            "localized_json/ro",
+            "localized_json/ru/BY",
+            "localized_json/ru/GE",
+            "localized_json/ru/KG",
+            "localized_json/ru/KZ",
+            "localized_json/ru",
+            "localized_json/ru/UA",
+            "localized_json/si",
+            "localized_json/sk",
+            "localized_json/sl",
+            "localized_json/sq",
+            "localized_json/sq/ME",
+            "localized_json/sr/Latn/ME",
+            "localized_json/sr/Latn/RS",
+            "localized_json/sv/FI",
+            "localized_json/sv",
+            "localized_json/sw/Latn/KE",
+            "localized_json/ta",
+            "localized_json/te",
+            "localized_json/th",
+            "localized_json/tr/AM",
+            "localized_json/tr/AZ",
+            "localized_json/tr/CY",
+            "localized_json/tr",
+            "localized_json/uk",
+            "localized_json/ur/IN",
+            "localized_json/ur",
+            "localized_json/uz",
+            "localized_json/vi",
+            "localized_json/zh",
+            "localized_json/zh/Hans/MY",
+            "localized_json/zh/Hans/SG",
+            "localized_json/zh/Hant/HK",
+            "localized_json/zh/Hant/TW"
+        ];
+
+        for (var i=0; i<locales.length;i++) {
+            jsrf = new AppinfoJsonFile({
+                project: p,
+                pathName: undefined,
+                type: ajft,
+                locale: locales[i]
+            });
+            test.equal(jsrf.getLocalizedPath(locales[i]), expected[i]);
+        }
+        test.done();
+    },
+    testJSONResourceFileGetResourceFilePathsSimple: function(test) {
+        test.expect(10);
+        var locales = ["af-ZA","am-ET","ar-AE","ar-BH","ar-DJ","ar-DZ","ar-EG","ar-IQ",
+        "ar-JO","ar-KW"];
+
+        var expected = [
+            "localized_json/af",
+            "localized_json/am",
+            "localized_json/ar/AE",
+            "localized_json/ar/BH",
+            "localized_json/ar/DJ",
+            "localized_json/ar/DZ",
+            "localized_json/ar",
+            "localized_json/ar/IQ",
+            "localized_json/ar/JO",
+            "localized_json/ar/KW",
+        ];
+
+        for (var i=0; i<locales.length;i++) {
+            jsrf = new AppinfoJsonFile({
+                project: p,
+                pathName: undefined,
+                type: ajft,
+                locale: locales[i]
+            });
+            test.equal(jsrf.getLocalizedPath(locales[i]), expected[i]);
+        }
+        test.done();
+    }
+};
```

Comment:
Need more tests to test parsing the files, localizing the text of the files, reading files from disk, and writing localized files to disk and making sure only localizable strings with translations are in the output.